python/pyarrow/array.pxi

```diff
@@ -1508,6 +1508,30 @@ cdef class DurationArray(NumericArray):
     Concrete class for Arrow arrays of duration data type.
     """
 
+cdef class MonthDayNanoIntervalArray(Array):
+    """
+    Concrete class for Arrow arrays of interval[MonthDayNano] type.
+    """
+
+    def to_pylist(self):
+        """
+        Convert to a list of native Python objects.
+
+        is installed the objects will be
+        pd.tseries.offsets.DateOffset objects.  Otherwise they are
+        pyarrow.MonthDayNanoTuple objects.
```

Comment:
Same remark as for the corresponding scalar class. Also, it seems the docstring is a bit garbled?

Reply:
Should be fixed now (once I push latest set of commits.